### PR TITLE
Update Python API to produce v1 ops

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: python/pyngraph/node_factory.cpp [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    virtual void on_attribute(const std::string& name, std::string& value) override
+    {
+        if (m_attributes.contains(name))
+        {
+            value = m_attributes[name.c_str()].cast<std::string>();
+        }
+    }
+    virtual void on_attribute(const std::string& name, bool& value) override
+    {
+        if (m_attributes.contains(name))
```
Comment (tsocha) [inline python/pyngraph/node_factory.cpp]: Why this if is needed every where?
What happen when name is absent from `m_attributes`?

File: python/setup.py [hunk 2]
```diff
@@ -102,6 +102,16 @@ def parallelCCompile(
     macros, objects, extra_postargs, pp_opts, build = self._setup_compile(
         output_dir, macros, include_dirs, sources, depends, extra_postargs)
     cc_args = self._get_cc_args(pp_opts, debug, extra_preargs)
+
+    if NGRAPH_PYTHON_DEBUG in ['TRUE', 'ON', True]:
+        try:
+            # pybind11 is much more verbose without -DNDEBUG
```
Comment (tsocha) [inline python/setup.py]: That's weird.

File: python/pyngraph/node_factory.cpp (hunk 1 above)
Comment (arogowie-intel) [reply python/pyngraph/node_factory.cpp]: We have to check whether the requested attribute name is present in the dictionary we received. If the attribute name is absent we just don't update it.

File: python/setup.py (hunk 2 above)
Comment (arogowie-intel) [reply python/setup.py]: But unfortunately that's true. When I made sure that this flag is present, then I much faster managed to find out what the problem was. That's just debugging convenience.

**Pull request merged**